src/safeds/data/tabular/containers/_table.py

```diff
@@ -1966,17 +1974,22 @@ def plot_histograms(self) -> Image:
         """
         col_wrap = min(self.number_of_columns, 3)
 
-        data = pd.melt(self._data, value_vars=self.column_names)
-        grid = sns.FacetGrid(data=data, col="variable", col_wrap=col_wrap, sharex=False, sharey=False)
-        grid.map(sns.histplot, "value")
-        grid.set_xlabels("")
-        grid.set_ylabels("")
-        grid.set_titles("{col_name}")
-        for axes in grid.axes.flat:
-            axes.set_xticks(axes.get_xticks())
-            axes.set_xticklabels(axes.get_xticklabels(), rotation=45, horizontalalignment="right")
-        grid.tight_layout()
-        fig = grid.fig
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                message="Converting input from bool to <class 'numpy.uint8'> for compatibility.",
+            )
+            data = pd.melt(self._data, value_vars=self.column_names)
+            grid = sns.FacetGrid(data=data, col="variable", col_wrap=col_wrap, sharex=False, sharey=False)
+            grid.map(sns.histplot, "value")
+            grid.set_xlabels("")
+            grid.set_ylabels("")
+            grid.set_titles("{col_name}")
+            for axes in grid.axes.flat:
+                axes.set_xticks(axes.get_xticks())
+                axes.set_xticklabels(axes.get_xticklabels(), rotation=45, horizontalalignment="right")
+            grid.tight_layout()
+            fig = grid.fig
```

Comment:
```suggestion
        data = pd.melt(self._data.applymap(lambda value: str(value)), value_vars=self.column_names)
        grid = sns.FacetGrid(data=data, col="variable", col_wrap=col_wrap, sharex=False, sharey=False)
        grid.map(sns.histplot, "value")
        grid.set_xlabels("")
        grid.set_ylabels("")
        grid.set_titles("{col_name}")
        for axes in grid.axes.flat:
            axes.set_xticks(axes.get_xticks())
            axes.set_xticklabels(axes.get_xticklabels(), rotation=45, horizontalalignment="right")
        grid.tight_layout()
        fig = grid.fig
```
For `Table.plot_histograms` maybe we should replace all the data with strings. This would fix the warning and it also would make the plot look a lot better.
An example with the table `Table({"a": [1, 2, 3, 3, 2, 4, 3], "b": [True, True, False, True, False, False, True], "c": ["a", "b", "b", "b", "b", "b", "b"]})`:
Before:
![original](https://github.com/Safe-DS/Stdlib/assets/47296670/74ba06b2-c8a7-483e-bc12-5fc472d6705e)
And after:
![all values as strings](https://github.com/Safe-DS/Stdlib/assets/47296670/2aee70a5-8e0f-413f-bcf9-ab24e7d51eed)

Reply:
This would also fix an error that is raised when you use a table with a column that contains boolean values and `None` values